src-plugins/itkFilters/itkFiltersBinaryCloseProcess_p.h

```diff
@@ -72,8 +72,8 @@ class itkFiltersBinaryCloseProcessPrivate : public itkMorphologicalFiltersProces
         QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
 
         if (isRadiusInPixels)
-            newSeriesDescription += " Bin.Close filter\n("+ QString::number(radius[0])+", "+
-            QString::number(radius[1])+", "+ QString::number(radius[2])+" pixels)";
+            newSeriesDescription += " Bin.Close filter\n("+ QString::number(floor(radius[0]))+", "+
+            QString::number(floor(radius[1]))+", "+ QString::number(floor(radius[2]))+" pixels)";
```

Comment:
If you are asking for a closing of X pixels, it means at most X pixels. So 5.99 would mean "less than 6". Rounding upwards means you will remove noise bigger than expected (it might not seem important to go from 5.99 to 6, but going from 1.5 to 2 is a big difference). If we allow a double value for a pixel radius, I think truncating is better, but restricting the input to an integer radius is even better.